DEPRECATE: hide-small-planets plugin

manavgarg opened the pull request:
Just a day after this plugin was updated to 0.6, the following commit in the client: https://github.com/darkforest-eth/client/commit/f13ca37a82be9f886cf49dd5980c85f3cf1c1a10 broke this plugin as the corresponding utils to get/setDetailLevel is no longer exposed.
Given the recent optimizations in the client which essentially hides smaller planets automatically, this plugin is no longer needed.
I would propose deprecating this plugin (currently have marked the version as [DEPRECATED]). This can then be deleted in the future.

phated commented:
@manavgarg This makes sense, but we should just add another metadata field called "deprecated" that is a boolean and change the template file to filter deprecated plugins.

manavgarg commented:
@phated Thanks for the suggestion. As per your feedback, I have introduced a new field and also added the condition to filter out all deprecated plugins.
I have tested this change locally by bringing up the site using the local hugo server.

phated reviewed (approved):
Looks good! Thanks!

phated commented on content/casual/hide-small-planets/index.md:
```diff
@@ -1,6 +1,7 @@
 ---
 title: Hide Small Planets
-date: 2021-01-16
+date: 2021-06-13
```
```suggestion
date: 2021-01-16
```
Revert date, as this is supposed to be submitted date.
